lib/src/user.dart

```diff
@@ -172,7 +172,8 @@ class User extends Event {
   /// Whether the client is allowed to change the power level of this user.
   /// Please be aware that you can only set the power level to at least your own!
   bool get canChangePowerLevel =>
```

Comment:
td took way too long to understand what this was supposed to do, can we change this to `canChangeUserPowerLevel` or something better?

Reply:
Okay changed :)